pkg/pillar/zedcloud/send.go

```diff
@@ -35,6 +35,9 @@ import (
 // ContentTypeProto : binary-encoded Protobuf content type
 const ContentTypeProto = "application/x-proto-binary"
 
+// MaxWaitForRequests : upper limit of time to send requests
```

Comment:
Would make sense to expand the comment to say "independent of how many management interfaces and source IP addresses we try".

Reply:
modified
also I put log for the case of global deadline